circle.yml

```diff
@@ -4,7 +4,7 @@ checkout:
     - git submodule update --init --recursive
 machine:
   node:
-    version: 0.12
+    version: 6.9.0
```

Comment:
If we want to use `6` as the baseline, this could just be `6` and it will automatically use the latest `6`.  However, since Meteor never shipped with an 8.x version I think it's safe to bump this to `8`.  I'll adjust accordingly.